Makefile.custom.mk

```diff
@@ -1,5 +1,6 @@
 .PHONY: generate
 generate:
+	rm -rf helm/cluster-api/templates/*
```

Comment:
We will have `crd-install` templates in there as well. I used this:

```
@rm -rf helm/cluster-api/templates/*.yaml
```

However I can update this when I merge CRDs into `release-1.0`.

Reply:
`-r` is unnecessary in this case I think.

Reply:
ok, let's fix those issues when adding the crds